zproject/urls.py

```diff
@@ -543,6 +544,7 @@
     rest_path("export/realm", POST=export_realm, GET=get_realm_exports),
     rest_path("export/realm/<int:export_id>", DELETE=delete_realm_export),
     rest_path("export/realm/consents", GET=get_users_export_consents),
+    # For unauth'd users
```

Comment:
This line seems unnecessary.

Reply:
Ah, yeah -- that was a holdover from when the `json/antispam_challenge` path was in there.